src/app/chat/chat-message/chat-message.component.scss

```diff
@@ -24,7 +24,7 @@ div {
 
   &.external-message {
     background-color: #d6d6d6;
-    float: left;
+    // float: left;
```

Comment:
Scroller doesn't like [margin](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Mastering_margin_collapsing) and float; its inner calculations are very minimalistic and don't cover all the possible cases that can be implemented as the CSS level. The best way would be to use another approach there (like padding instead of margin).

Reply:
thanks for the tip!